clickhouse/values.yaml

```diff
@@ -24,6 +24,8 @@ client:
 
 # zookeeper
 zookeeper:
+  repository: gcr.io/google_samples/k8szk:v1
```

Comment:
Please do not use this image.

Images in gcr.io/google-samples are:

* meant to be used only in GKE tutorials and samples
* not patched/rebuilt actively with security fixes etc
* not actively maintained in general
* should not be used for production purposes
* there no guarantees whether they simply won't be deleted tomorrow.

Reply:
@spaghettifunk do you know what is the official image which can work with K8S?

Reply:
@plutov  I would use `"k8s.gcr.io/kubernetes-zookeeper:1.0-3.4.10"`. Check the [Clickhouse-Operator](https://github.com/Altinity/clickhouse-operator/blob/master/manifests/zookeeper/advanced/05-stateful-set.yaml) from Altinity.

Reply:
I am trying this image bu container fails with this error:
```     
sh: 1: zkGenConfig.sh: not found```

Reply:
check this [repo](https://github.com/kubernetes/contrib/tree/master/statefulsets/zookeeper). I think it comes from here.